src/app/cube/usage-stats/usage-stats.component.ts

```diff
@@ -121,6 +125,54 @@ export class UsageStatsComponent implements OnInit {
       ];
   }
 
+  /**
+   * Constructs chart for Organization Breakdown distribution
+   *
```

Comment:
We should add a fix me to the docs here so that it’s clear that this data is hardcoded and should eventually be pulled from our backend so that it can update dynamically